scripts/python-wheels-windows.bat

```diff
@@ -80,23 +79,25 @@ pushd parquet-cpp\build
 cmake -G "%GENERATOR%" ^
      -DCMAKE_INSTALL_PREFIX=%PARQUET_HOME% ^
      -DCMAKE_BUILD_TYPE=Release ^
-     -DPARQUET_BOOST_USE_SHARED=off ^
-     -DPARQUET_BUILD_TESTS=off .. || exit /B
+     -DPARQUET_BOOST_USE_SHARED=ON ^
+     -DPARQUET_BUILD_TESTS=OFF .. || exit /B
 cmake --build . --target INSTALL --config Release || exit /B
 popd
 
 @rem Build and import pyarrow
 set PYTHONPATH=
 
 pushd %ARROW_SRC%\python
-python setup.py build_ext --with-parquet --bundle-arrow-cpp bdist_wheel  || exit /B
+python setup.py build_ext --with-parquet ^
+    --bundle-arrow-cpp --bundle-boost bdist_wheel || exit /B
```

Comment:
@xhochy Is it possible to bundle boost without building from source and putting it into a custom namespace?

Reply:
Yes it is

Reply:
Great!